### PR TITLE
Add methods for create and delete events

### PR DESCRIPTION
**Pull request by kozlovpn**
Add methods for create and delete events

### CODE REVIEW

File: shellper/main.py [hunk 1]
```diff
@@ -19,14 +19,15 @@ def main():
...
     google = base.Base()
 
     google.get_event_list()
-
-    links = []
     for event in config:
-        links.append(google.search_query(event["summary"]))
+        event["description"] = []
+        event["description"].append(google.search_query(event["summary"]))
+    for event in config:
```
Comment (ivanovmi) [inline shellper/main.py]: Why you are using duplicate `for event in config`? What does it mean?

Comment (kozlovpn) [inline shellper/main.py]: @ivanovmi in first for event in config we add new field description to config with array of links. And in second for event in config we create events from config and then delete the same events

Comment (esikachev) [inline shellper/main.py]: @kozlovpn +!

**Comment by ivanovmi** [comment]
+2

**Pull request merged**